app/assets/images/wifi_icon.svg

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Why would we want to use xml here?

Reply:
SVG is a subset of XML. All SVG images are stored as XML documents and it's not uncommon to hand-edit the SVG document and/or render it dynamically via JSX just like we do with HTML documents.

This particular line is roughly equivalent to the `<!doctype html>` line that starts HTML documents. Version 1.0 is the default value, so in XML documents it can be (and usually is) omitted. Inkscape apparently includes it, though. ¯\\_(ツ)_/¯